app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,2 +1,13 @@
+$primary: #1a454a;
+$secondary: rgb(241, 240, 240);
+
 @import 'bootstrap/scss/bootstrap';
 @import 'bootstrap-icons/font/bootstrap-icons';
+
+// TODO: Hadi- Move this to specific file to only be accessible where needed
+#rooms-card {
+  &:hover {
+      cursor: pointer;
+      background: rgba(26, 69, 74, 0.2)
```

Comment:
Can we declare a variable and use it here with the provided color like with what you have done above?

Reply:
`rooms-card` will be used by several room cards.
It should be a `class` rather then an `id`